x-pack/plugins/security_solution/server/usage/detections/rules/get_metrics.mocks.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SearchResponse } from '@elastic/elasticsearch/lib/api/types';
+
+export const getMockRuleAlertsResponse = (docCount: number): SearchResponse<unknown, unknown> => ({
+  took: 7,
+  timed_out: false,
+  _shards: {
+    total: 1,
+    successful: 1,
+    skipped: 0,
+    failed: 0,
+  },
+  hits: {
+    total: {
+      value: 7322,
+      relation: 'eq',
+    },
+    max_score: null,
+    hits: [],
+  },
+  aggregations: {
+    buckets: {
+      after_key: {
+        detectionAlerts: '6eecd8c2-8bfb-11eb-afbe-1b7a66309c6d',
+      },
+      buckets: [
+        {
+          key: {
+            detectionAlerts: '6eecd8c2-8bfb-11eb-afbe-1b7a66309c6d',
+          },
+          doc_count: docCount,
+        },
+      ],
+    },
+  },
+});
+
+export const getMockAlertCasesResponse = () => ({
```

Comment:
Same question for mocks for alerting / cases. We want to let upstream developers know they have bust us somehow with changes to their code.

Reply:
I will check this one though and see I can give it a stronger type for you.

Reply:
I was able to get a stronger type as suggested and it's from the Case team. It will be now:

```ts
export const getMockAlertCaseCommentsResponse = (): SavedObjectsFindResponse<
  Partial<CommentAttributes>,
  never
> => ({
```

I fixed the other types in this file to be stronger as well 👍